data-sources/activemq/config.yml

```diff
@@ -0,0 +1,12 @@
+id: activemq
```

Comment:
This data source already exists as a `core` data source. You can delete this file

Reply:
done